lib/src/action-icon/ActionIcon.test.tsx

```diff
@@ -10,22 +10,22 @@ const iconSVG = (
 );
 describe("Action icon component tests", () => {
   test("Action icon renders with correct text", () => {
-    const { getByTitle } = render(<DxcActionIcon icon={iconSVG} title="favourite" />);
-    expect(getByTitle("favourite")).toBeTruthy();
+    const { getByTestId } = render(<DxcActionIcon icon={iconSVG} title="favourite" />);
```

Comment:
Avoid using dataTestId if possible.
For more info:
https://testing-library.com/docs/queries/about/#priority

Reply:
Fixed.